app/views/trainees/check_details/show.html.erb

```diff
@@ -1,3 +1,13 @@
+<%= content_for(:breadcrumbs) do %>
```

Comment:
Missing page title for browser

![image](https://user-images.githubusercontent.com/470137/99811762-7cb26e80-2b3d-11eb-9aaf-daf43be28b01.png)


aka  
```ruby
<%= render_component PageTitle::View.new(title: "check_details.show") %>
```

Reply:
done thanks https://github.com/DFE-Digital/register-trainee-teachers/pull/193/commits/4885434fa4c27dfce192646a71ea3c2e54b46fb0